opentreemap/api/plots.py

```
@@ -72,7 +71,16 @@ def update_or_create_plot(request, instance, plot_id=None):
     # The API communicates via nested dictionaries but
     # our internal functions prefer dotted pairs (which
     # is what inline edit form users)
-    request_dict = json.loads(request.body)
+    # Sometimes reading from body fails, so try reading as a file-like object
+    try:
+        body_decoded = request.body.decode()
+    except:
```

It is generally preferable to catch an explicit exception type in these cases to avoid "swallowing" an exception that is unrelated to the body reading (`MemoryError` for example)